### ISSUE TITLE
one suggestion

### ISSUE THREAD

**Issue by jake1987** [marcosesperon/jquery.rowspanizer.js#1]
great job!!
maybe u can add some rows or colums like $('#table').rowspanizer({tr:[01,2...],td:[3,4]});

**Comment by VelizarVESSELINOV** [comment]
I agree that the option select columns and exclude some rows will be a great addition.
Example: rowspanizer on columns selection first two and last two, excluding last 10 rows

and if possible do colspanizer on the last 10 rows will be a nice addition.
